doc/tutorials/content/gpu_install.rst

```diff
@@ -2,85 +2,44 @@
 
 Configuring your PC to use your Nvidia GPU with PCL
 ---------------------------------------------------
-In this tutorial we will learn how to check if your PC is 
-suitable for use with the GPU methods provided within PCL.
-This tutorial has been tested on Ubuntu 11.04 and 12.04, let
-us know on the user mailing list if you have tested this on other
-distributions.
+In this tutorial you will learn how to make your configure your system to make it compatible to run the GPU methods provided by PCL.
+This tutorial is for Ubuntu, other Linux distrubutions can follow a similar process to set it up.  
 
-The explanation
----------------
-
-In order to run the code you'll need a decent Nvidia GPU with Fermi or Kepler architecture you can check this by::
-
- $ lspci | grep nVidia
-
-This should indicate which GPU you have on your system, if you don't have an Nvidia GPU, we're sorry, but you
-won't be able to use PCL GPU.
-The output of this you can compare with `this link <http://www.nvidia.co.uk/object/cuda-parallel-computing-uk.html>`_  
-on the Nvidia website, your card should mention compute capability of 2.x (Fermi) or 3.x (Kepler) or higher.
-If you want to run with a GUI, you can also run::
-
- $ nvidia-settings
-
-From either CLI or from your system settings menu. This should mention the same information.
-
-First you need to test if your CPU is 32 or 64 bit, if it is 64-bit, it is preferred to work in this mode.
-For this you can run::
-
-  $ lscpu | grep op-mode
-
-As a next step you will need a up to date version of the Cuda Toolkit. You can get this 
-`here <http://developer.nvidia.com/cuda/cuda-downloads>`_, at the time of writing the
-latest version was 4.2 and the beta release of version 5 was available as well.
-
-First you will need to install the latest video drivers, download the correct one from the site
-and run the install file, after this, download the toolkit and install it.
-At the moment of writing this was version 295.41, please choose the most up to date one::
-
-  $ wget http://developer.download.nvidia.com/compute/cuda/4_2/rel/drivers/devdriver_4.2_linux_64_295.41.run
-
-Make the driver executable::
-
- $ chmod +x devdriver_4.2_linux_64_295.41.run
-
-Run the driver::
+Windows is nto currently officialy supported for the GPU methods.
 
- $ sudo ./devdriver_4.2_linux_64_295.41.run
-
-You need to run this script without your X-server running. You can shut your X-server down as follows:
-Go to a terminal by pressing Ctrl-Alt-F1 and typing::
+The Explanation
```

Comment:
Maybe rename to `Installing dependencies` and add a section for CUDA and another for Eigen

Reply:
I added a few sections. Is that fine?

Reply:
Could you also mention that code works with clang (9 (or whatever is on 18.04), min version unknown) and GCC (min version 7) instead of saying works only with GCC 7 or later

Reply:
Added